task/power/src/bsp/cosmo_a.rs

```diff
@@ -135,6 +135,12 @@ enum Trace {
         err: drv_i2c_api::ResponseCode,
     },
 
+    /// Calling `CLEAR_FAULTS` on the LM5066 failed
+    Lm5066ClearFaultsFailed {
+        index: usize,
+        err: drv_i2c_api::ResponseCode,
+    },
+
```

Comment:
Should this be LM5066i (and a couple of other places)

Reply:
That's done by the next PR in the stack (#2076); this one doesn't know about the alternate chip.

Reply:
Ahh somehow I mixed up the order